h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -1694,7 +1694,7 @@ private TwoDimTable makeCrossValidationSummaryTable(Key[] cvmodels) {
       stats.add(vals[i],1);
     for (i=0;i<numMetrics;++i) {
       table.set(i, 0, (float)stats.mean()[i]);
-      table.set(i, 1, (float)stats.sigma()[i]);
+      table.set(i, 1, (float)stats.sigmaNorm()[i]);
```

Comment:
@wendycwong could we just use `org.apache.commons.math3.stat.descriptive.moment.StandardDeviation` instead?

We are just using few values here (no big data), imho implemeting this in the framework of BasicStats is not necesasary.

Reply:
The problem here is I like the BasicStats framework.  They take care of NA counts, weighted means.  I just added two fields there, _varianceNorm, _sigmaNorm.  If the functions varianceNorm() or sigmaNorm() are not called, the two new fields will be null.